src/extension.ts

```diff
@@ -24,7 +24,8 @@ export function activate(context: vscode.ExtensionContext) {
 			const bazelOptions = ''; // or '--config=debug'
 			const ruleName = 'specs'; // TODO: add an option to specify the rule name.
 			const prefixSpace = configuration.noHistory ? ' ' : '';
-			const consoleCommand = `${prefixSpace}JAS_TARGET=$(bazel query ${pathToFile}); ${bazelType} ${bazelCommand} \${JAS_TARGET%:*}:${ruleName} ${bazelOptions} --test_filter="\${JAS_TARGET#*:}${(spec.specFilter ? '#' + spec.specFilter : '')}"`;
+			const maybeMoveToGitRoot = configuration.cdGitRoot ? 'cd $(git rev-parse --show-toplevel); ' : '';
```

Comment:
Instead of `cd` to git root, how hard do you think it would be to modify the other commands to strip extra parts of the paths?

Reply:
I think it would just be a matter of getting `pwd` from the current console and having the `pathToFile` be `path.relative(pwd, pathToFile)` instead of using the `pathToWorkspace` as the base directory.

I'm not sure if there is a way to get the cwd from a terminal that already exists though.

Alternative is to not use git at all and do
```
cd ${workspace.uri};
```
at the beginning of every command.

Reply:
I think there's a problem with the "cd to git root" idea if the workspace root isn't the git root then the relative path we're sending would be wrong.